bluemix/terminal/table.go

```diff
@@ -69,58 +76,160 @@ func (t *PrintableTable) Add(row ...string) {
 	}
 }
 
+func isWideColumn(col string) bool {
+	// list of common columns that are usually wide
+	largeColumnTypes := []string{T("ID"), T("Description")}
+
+	for _, largeColn := range largeColumnTypes {
+		if strings.Contains(largeColn, col) {
+			return true
+		}
+	}
+
+	return false
+
+}
+
+func terminalWidth() int {
+	var err error
+	terminalWidth, _, err := term.GetSize(int(os.Stdin.Fd()))
+
+	if err != nil {
+		// Assume normal 80 char width line
+		terminalWidth = 80
+	}
+
+	testTerminalWidth, envSet := os.LookupEnv("TEST_TERMINAL_WIDTH")
```

Comment:
**Callout**: Added test environment variable to test variable terminal width in unit tests